src/ui/Facet/FacetSearchElement.ts

```diff
@@ -281,6 +282,7 @@ export class FacetSearchElement {
     this.combobox.setAttribute('role', 'combobox');
     this.combobox.setAttribute('aria-owns', this.facetSearchId);
     this.input.setAttribute('aria-controls', this.facetSearchId);
+    this.input.setAttribute('aria-expanded', true.toString());
```

Comment:
I'd prefer `true` as a string directly ;)

Reply:
Alright, changing it to `'true'`.